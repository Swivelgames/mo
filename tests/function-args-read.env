testArgs() {
    echo "${MO_FUNCTION_ARGS[*]}"
}
