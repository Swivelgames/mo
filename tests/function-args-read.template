No args: [{{testArgs}}] - done
One arg: [{{testArgs "one"}}] - done
Multiple arguments: [{{testArgs "aa" "bb" "cc" "'x'" "\" ! {[_.|" }}] - done
Evil: [{{testArgs "bla; cat /etc/issue"}}] - done
