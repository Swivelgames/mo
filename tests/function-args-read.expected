No args: [] - done
One arg: [one] - done
Multiple arguments: [aa bb cc 'x' " ! {[_.| ] - done
Evil: [bla; cat /etc/issue] - done
